iModelCore/ECDb/ECDb/ECSql/ECSqlParser.cpp

```diff
@@ -1899,6 +1899,47 @@ BentleyStatus ECSqlParser::ParseTableValuedFunction(std::unique_ptr<TableValuedF
     exp = std::make_unique<TableValuedFunctionExp>(schemaName.c_str(), std::move(memberFuncCall), PolymorphicInfo::All());
     return SUCCESS;
 }
+
+//-----------------------------------------------------------------------------------------
+// @bsimethod
+//+---------------+---------------+---------------+---------------+---------------+------
+BentleyStatus ECSqlParser::ParseTableNodeRef(std::unique_ptr<ClassRefExp>& exp, OSQLParseNode const& tableNode, ECSqlType ecsqlType) const

```

Comment:
Since ECSqlParser is very big, I do not like this method name. It's so vague. That is a tableNodeRef?
But I see the other method is already called ParseTableNode, mmh....
If this turns out to be only used in relationships, maybe at least call it ParseRelationshipJoinTableNode, that would add a little context

Reply:
Hm closing this comment. After reviewing the other method names I don't want to break the existing scheme.

Reply:
I wasn't sure what to call it and as ParseTableNode(...) made a prt<ClassNameExp> and I needed a ClassRefExp  to store the table node alias, I called the function ParseTableNodeRef(...)